recipe/meta.yaml

```diff
@@ -32,10 +31,7 @@ requirements:
     # optional deps [brotli]
     - brotli-python >=1.0.9
   run_constrained:
-    # based on https://github.com/urllib3/urllib3/issues/2680
```

Comment:
We should leave those run_constrained. What is described in the issue is still valid.